loopchain/blockchain/blockchain.py

```diff
@@ -98,6 +99,8 @@ def __init__(self, channel_name=None, store_id=None, block_manager=None):
         for tx_version, tx_hash_version in channel_option.get("hash_versions", {}).items():
             self.__tx_versioner.hash_generator_versions[tx_version] = tx_hash_version
 
+        self._init_blockchain()
```

Comment:
I like this line!
Good for init object when initialized, not in the other classes! 
It helps increasing readability I think.